Resources/public/js/alloyeditor/buttons/linkedit.js

```diff
@@ -312,15 +312,23 @@ YUI.add('ez-alloyeditor-button-linkedit', function (Y) {
             var editor = this.props.editor.get('nativeEditor'),
                 linkUtils = new CKEDITOR.Link(editor),
                 selection = editor.getSelection(),
-                bookmarks = selection.createBookmarks();
+                bookmarks = selection.createBookmarks(),
+                scrollY;
 
             linkUtils.remove(this.state.element, {advance: true});
 
             selection.selectBookmarks(bookmarks);
 
             this.props.cancelExclusive();
 
-            editor.fire('actionPerformed', this);
+            if (navigator.userAgent.indexOf('Chrome') > -1) {
+                // Workaround for https://jira.ez.no/browse/EZP-28565
+                scrollY = window.pageYOffset;
+                editor.fire('actionPerformed', this);
```

Comment:
I'd move this line outside the `if` clause, because it's fired in both cases.

Reply:
This is from branch merge, so should not be fixed here, but rather by @adamwojs on original affected 1.x branch (PR was: #935)

Reply:
I disagree if the assumption that we apply this workaround only for Chrome (made in https://github.com/ezsystems/PlatformUIBundle/pull/920#issuecomment-344910360) is still valid.